lntest/itest/test_harness.go

```diff
@@ -31,7 +31,7 @@ var (
 		"lndexec", itestLndBinary, "full path to lnd binary",
 	)
 
-	slowMineDelay = 50 * time.Millisecond
+	slowMineDelay = 20 * time.Millisecond
```

Comment:
re commit message: why does decreasing this value slow things down?

Reply:
We already had the `mineBlocksSlow` function that used the 50ms delay. By replacing **all** instances of `mineBlocks` with `mineBlocksSlow`, we make slow everything down. To reduce the amount of overall slowdown we decrease the delay from 50ms to 20ms.

Reply:
Going to update the commit message to make this more clear.